modules/Cargo.toml

```diff
@@ -17,13 +17,15 @@ paths-cosmos = []
 paths-ics = []
 
 [dependencies]
-tendermint = { git = "https://github.com/informalsystems/tendermint-rs.git", branch = "master" }
+tendermint = { path = "/Users/ancaz/rust/tendermint-rs/tendermint"}
```

Comment:
- you can use a relative path here instead of the absolute one
- iirc one can specify both `git` and `path`, and Cargo will prioritize the `path` dependency over `git` if it exists

Reply:
thanks, will keep that in mind. I switched back to git as my local changes were just committed.